update.py

```diff
@@ -219,6 +226,65 @@ def copy_build(site):
         shutil.rmtree(olddir)
 
 
+def copy_and_keep_build(site):
+    """
+    Copies each site into the target location and keep last "n" builds as backups
+    """
+    global error_count
+    for wiki in ALL_WIKIS:
+        if site=='common':
+            continue
+        if site is not None and site != wiki:
+            continue
+        debug('copy: %s' % wiki)
+        targetdir = os.path.join(args.destdir, wiki)
+
+        debug("Creating temporary folders")
+        previousdir = os.path.join(args.backupdestdir)
+        debug('Recreating %s' % previousdir )
+        if not os.path.exists(previousdir):
+            os.mkdir(previousdir)
+    
+        olddir = os.path.join(previousdir, str(building_time) )
+        debug('Recreating %s' % olddir )
+        if not os.path.exists(olddir):
+            os.mkdir(olddir)
```

Comment:
Consider using
```suggestion
        olddir = os.path.join(args.backupdestdir, str(building_time) )
        debug('Recreating %s' % olddir )
        distutils.dir_util.mkpath(olddir);
```

Reply:
Sure. Updated ln 264 due to args.backupdestdir